From c341a3e75a4d79d9f00cfdf998dbfb7db3aac947 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Wed, 7 Jun 2023 09:05:09 +0200
Subject: [PATCH 001/145] Revert "btrfs: call btrfs_orig_bbio_end_io in
 btrfs_end_bio_work"

This reverts commit afdaf47f3993cf73d6a7925040c791ec008024c3.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 fs/btrfs/bio.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/btrfs/bio.c b/fs/btrfs/bio.c
index ada899613..726592868 100644
--- a/fs/btrfs/bio.c
+++ b/fs/btrfs/bio.c
@@ -307,7 +307,7 @@ static void btrfs_end_bio_work(struct work_struct *work)
 
 	/* Metadata reads are checked and repaired by the submitter. */
 	if (bbio->bio.bi_opf & REQ_META)
-		btrfs_orig_bbio_end_io(bbio);
+		bbio->end_io(bbio);
 	else
 		btrfs_check_read_bio(bbio, bbio->bio.bi_private);
 }
-- 
2.41.0.rc2

